vassal-app/src/main/java/VASSAL/build/module/Console.java

```diff
@@ -59,6 +59,23 @@ private boolean matches(String s1, String s2, int min) {
     return s1.toLowerCase().substring(0, s2.length()).equals(s2.toLowerCase());
   }
 
+  /**
+   * Opens the specified file (or folder) using the Desktop's default method
+   * @param file file or folder to open
+   */
+  private void browseFileOrFolder(File file) {
+    final Desktop desktop = Desktop.getDesktop();
+    try {
+      desktop.open(file);
+    }
+    catch (IOException e) {
+      System.out.println("File Not Found"); //NON-NLS
+    }
+    catch (IllegalArgumentException iae) {
+      System.out.println("Illegal argument"); //NON-NLS
```

Comment:
One more: Shouldn't these go to the logger instead of stdout?

Reply:
Probably & changed (even though wherever in VASSAL I copied that snippet from was doing it that way - I think we're rife with it)